scripts/kind-e2e/lib_armada_deploy_kind.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# This should only be sourced
+if [ "${0##*/}" = "lib_helm_deploy_subm.sh" ]; then
+    echo "Don't run me, source me" >&2
+    exit 1
+fi
+
+### Variables ###
+
+# FIXME: Armada support for setting pod/service CIRDs via flags?
+declare -A cluster_CIDRs=( ["cluster1"]="10.4.0.0/14" ["cluster2"]="10.8.0.0/14" ["cluster3"]="10.12.0.0/14" )
+declare -A service_CIDRs=( ["cluster1"]="100.1.0.0/16" ["cluster2"]="100.2.0.0/16" ["cluster3"]="100.3.0.0/16" )
+
+kubecfgs_rel_dir=scripts/output/kube-config/container/
+kubecfgs_dir=${PRJ_ROOT}/$kubecfgs_rel_dir
+
+### Functions ###
+
+function install_armada() {
+    # FIXME: Armada will be installed in dapper-base:latest after this patch, making this fn unnecessary.
+    # FIXME: Remove this function after CI validates the rest of the patch
+    test -x /usr/bin/armada && return
+    # FIXME: Use a release from GitHub once one is available
+    curl -L https://drive.google.com/u/0/uc\?id\=17r6E0RwbHcvlaFl0LKFICuAc_eGb13W9\&export\=download \
+         -o /usr/bin/armada
+    chmod a+x /usr/bin/armada
+}
+
+function create_kind_clusters() {
+    # FIXME: Remove this function after CI validates the rest of the patch
+    install_armada
+
+    # FIXME: Install tiller when deploying with helm
+    /usr/bin/armada create clusters -n 3 --weave
+}
+
+function import_subm_images() {
+    docker tag quay.io/submariner/submariner:dev submariner:local
```

Comment:
Do you really need to re-tag them? Can't armada just load the original image tags?

Reply:
I think the deployment expects to find :local images. It's also an indicative we use to know that the deployments should not "pull always" (subctl detects that, and we force it into helm I think)

Reply:
I think the deployment tools expect the :local tag when we don't want to re-pull a remote image.